aspnetcore/signalr/configuration.md

```diff
@@ -57,6 +57,7 @@ The following table describes options for configuring SignalR hubs:
 
 | Option | Default Value | Description |
 | ------ | ------------- | ----------- |
+| `ClientTimeoutInterval` | 30 seconds | The server will consider the client disconnected if it hasn't received a message (including keep-alive) in this interval. It is possible for it to take longer than this timeout interval for the client to actually be marked disconnected, due to how this is implemented. |
```

Comment:
Could simplify a bit -- It is possible for it to --> It might

Could delete "due to how this is implemented"